data_structures/trie/C++/trie_impl.cpp

```diff
@@ -0,0 +1,56 @@
+#include<bits/stdc++.h>
```

Comment:
Does this file relate to the problem at all?

Although we cannot accept the previous file, perhaps we could change your contribution to focus on this `trie` structure implementation... what do you think? 😸